src/model/saveScore.js

```diff
@@ -0,0 +1,29 @@
+export const saveScore = (gameType, userName, userScore) =>{
+
+    let scoreTable = getScores(gameType);
+
+    const scoreObject = {
+        'name': userName, 
+        'score': userScore
+    };
+
+    if(!scoreTable){
+        scoreTable = [];
```

Comment:
Takie krótkie if'y możesz zastąpić krótszą wersją
```suggestion
    if(!scoreTable) scoreTable = [];
```